app/services/idv/vendor_validated.rb

```diff
@@ -1,21 +0,0 @@
-module Idv
-  module VendorValidated
-    extend ActiveSupport::Concern
-
-    def vendor_validate
```

Comment:
if we are not using this anymore, how are we ensuring that these "step" classes all have the same API?